packages/lib-classifier/src/components/Classifier/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -60,104 +57,104 @@ class SubTaskPopup extends React.Component {
     super()
     this.popup = React.createRef()
   }
-  
+
   close () {
     this.props.setSubTaskVisibility(false)
   }
-  
-  // TODO: Split render() into various asyncStates?
+
+  // TODO:
+  // - split render() into various asyncStates?
+  // - add TranscriptionTask text input
```

Comment:
I think you're right, and it's prob right in front of me I'm just not putting the pieces (TranscriptionTask, TranscriptionLine, <-- a related activeMark, etc.) together, but in the SubTaskPopup it gets the `tasks` to iterate through an render (i.e. for drawing task, maybe a question and text task) from `(activeMark && activeMark.tasks) ? activeMark.tasks : []` and I'm not sure how the text input related to the transcription task would be included in that?

Reply:
The `TextTask` is generic and will render an input with suggestions if `suggestions` prop is passed to it, so if the transcription task has a sub-task of a text task defined, under the task's `details` property, it'll render it like the drawing task would. Does that answer your question? It might be clearer once we get a mock task and story setup.